src/riscv-double-trap.adoc

```diff
@@ -240,7 +240,7 @@ re-entry into Debug Mode due to the critical error if `cetrig` is set to 1, or
 it will cause the critical-error signal to be asserted to the platform.  The
 debugger can resume with `cetrig` set to 0 to allow the platform-defined actions
 on critical-error signal to occur. Other possible actions include initiating a
-hart or platform by using the Debug Module reset control.
+hart or platform using the Debug Module reset control.
```

Comment:
@kersten1 
The sentence is not reading right. It says:
"Other possible actions include initiating a hart or platform using the Debug Module reset control."
It should say:
"Other possible actions include initiating a hart or platform reset using the Debug Module reset control."

Reply:
@kersten1 - I will merge this and fix this post merge.